packages/web-components/src/components/back-to-top/back-to-top.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import settings from 'carbon-components/es/globals/js/settings.js';
+import UpToTop20 from 'carbon-web-components/es/icons/up-to-top/20.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+// import './back-to-top-button';
```

Comment:
Did you mean to remove this or keep this...?

```suggestion
// import './back-to-top-button';
```